feature-libs/customer-ticketing/assets/translations/en/customer-ticketing.ts

```diff
@@ -15,5 +17,12 @@ export const customerTicketing = {
     uploadFile: 'Upload File',
     fileSizeLimit: 'File Size Limit: {{count}}MB',
     maximumAttachment: 'Maximum of one attachment per message',
+    noTickets: `You don't have any request`,
+    requestTitle: 'All Requests',
+    sortSubtitle: 'Sort by',
+    // Mobile view
```

Comment:
export const customerTicketing = {
 customerTicketingDetails: {
 anyKeyValueForThisPage
}

customerTicketingList: {
anyKeyvaluePair or even complex objects

desktop: {
 anyKeyValuePair
}

mobile: {
anyKeyValuePair
}
}

}

does that make sense?
Let's split this entire translations into chunks basically